autogpt_platform/frontend/src/components/edit/control/GraphSearchControl.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
```

Comment:
This is for the current system and can be removed when the new control panel becomes active